web/src/components/organisms/Asset/AssetList/hooks.ts

```diff
@@ -154,6 +177,14 @@ export default () => {
     navigate(`/workspace/${workspaceId}/project/${projectId}/asset/${asset.id}`);
   };
 
+  const hideUploadModal = useCallback(() => {
```

Comment:
handleUploadModalClose

Reply:
Done!

Reply:
I see you updated it to handleUploadModalCancel. I get that closing an update modal is kinda like "cancelling" the update, but in actual fact the callback is handling the close of the modal, right? So can you make it "close"

Reply:
I think you have updated other callbacks too with xxxCancel. Can you do the same for those too?